<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstellationSelectionPopupView</class>
 <widget class="QWidget" name="ConstellationSelectionPopupView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>109</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QCheckBox" name="beidouCheckBox">
     <property name="text">
      <string>BEIDOU</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="imesCheckBox">
     <property name="text">
      <string>IMES</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="sbasCheckBox">
     <property name="text">
      <string>SBAS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="gpsCheckBox">
     <property name="text">
      <string>GPS</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="qzssCheckBox">
     <property name="text">
      <string>QZSS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="glonassCheckBox">
     <property name="text">
      <string>GLONASS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="galileoCheckBox">
     <property name="text">
      <string>GALILEO</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="submitConstellationsButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
